contracts/Recruitment.sol

```diff
@@ -427,18 +436,21 @@ contract Recruitment is Ownable, ReentrancyGuard {
         jobList[_jobId].isDisbursed = true;
         uint256 bounty = jobList[_jobId].bounty;
 
-        bountyClaim[jobCandidatehire[_jobId].referrer] =
-            bountyClaim[jobCandidatehire[_jobId].referrer] +
-            (bounty * 6500) /
-            10_000;
-        bountyClaim[jobCandidatehire[_jobId].wallet] =
-            bountyClaim[jobCandidatehire[_jobId].wallet] +
-            (bounty * 1000) /
-            10_000;
+        uint hiredCount = hiredCpandidateListForJob[_jobId].length;
```

Comment:
confirm with @Hangout2698 if we can hire multiple candidates for one job, and if yes - how we want to split bounty??